content/en/docs/Univariate/MCMCglmm/_index.Rmd

```diff
@@ -9,10 +9,13 @@ output:
     keep_md: yes
 ---
 
+```{r setup_fig, include=FALSE}
+knitr::opts_chunk$set(fig.path = "plots/")
```

Comment:
In my understanding and local testing this does not work (at least not without a manual edit of .md files after compilation). I think it would be a better solution to save plots in "." instead of "plots/". That way Hugo sees the plots with the syntax as written in the .md files that are produced by knitr. 
But maybe you have a way to make Hugo (or maybe rather the Docsy theme) see inside the "plots/" folder?

Reply:
Yes you are right. I guess I tried with plots/ but forgot to knit the new file and though it was working. I will update with `fig.path = ""`, which do the trick nicely.